index.html

```diff
@@ -815,17 +815,26 @@ <h3>Bitstring Expansion Algorithm</h3>
 
   </section>
 
-  <section class="informative">
+  <section class="normative">
     <h2>Media Types</h2>
+
+    <p>
+<a>Issuers</a> SHOULD publish status list information
+using HTTPS URLs and in ways that minimize possible correlation of usage
+patterns related to the list. <a>Verifiers</a>
+SHOULD retrieve status list information using protocols that guard against access
+pattern correlation, such as Oblivious HTTP [[?OHTTP]].
+    </p>
+
     <p>
-  When dereferencing `statusListCredential`, the content type of the
-  `statusListCredential` might be any media type registered for the purpose of
-  expressing a verifiable credential with one or more proofs.
+When dereferencing `statusListCredential`, the content of the returned
+`statusListCredential` might be any media type registered for the purpose of
+expressing a verifiable credential with one or more proofs.
     </p>
     <p>
-  For example, a verifiable credential secured with Data Integrity Proofs might
-  have content type `application/vc+ld+json`, whereas a verifiable credential
-  secured with SD-JWT might have content-type `application/sd-jwt`.
+For example, a verifiable credential secured with Data Integrity Proofs might
+have media type `application/vc+ld+json`, while a verifiable credential
+secured with SD-JWT might have media type `application/sd-jwt`.
```

Comment:
```suggestion
secured with SD-JWT might have media type `application/vc+ld+json+sd-jwt`.
```

Reply:
This change is being challenged in the latest JOSE COSE spec.